### PR TITLE
Prevents ants from infesting wrapped/packaged food

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Adds `antable = FALSE` to the following foods:
**4no Raisins:**
Raisins
**Admiral Yamamoto Carp:**
Carp
**Bread Tube:**
Bread
**Candy:**
Candy
**Cheesie Honkers:**
Honkers
**Chinese Newdles:**
Newdles
**Chips:**
Chips
**Chow Mein:**
Chow mein
**Fried Rice:**
Fried rice
**Pistachios:**
Pistachios
**Scaredy's Private Reserve Beef Jerky:**
Jerky
**Syndi-Cakes:**
Syndicakes

(Same idea as one of the fixes in #16520, but expanded upon a bit.)

## Why It's Good For The Game

All of those foods have wrappers or containers, so it makes sense to me that ants wouldn't be able to get to them. I know I at least have been caught up in the past trying to eat some unopened 'Cheesie Honkers', only to find that ants somehow got through the packaging.
Basically, it's just less unintuitive.

## Changelog
:cl:
tweak: Prevented ants from getting into wrapped food from vending machines.
/:cl:

### CODE REVIEW

**Comment by meow20** [comment]
Antable. Best name ever Sabre, you should name everything in our code :D

**Review by SteelSlayer — APPROVED** [review]

**Comment by Wojz** [comment]
> Antable. Best name ever Sabre, you should name everything in our code :D

Oh that's been in the code for a while.

**Comment by meow20** [comment]
> Oh that's been in the code for a while.

I wonder how many other things in code have such derpy names lol

**Review by variableundefined — APPROVED** [review]
